src/web/app/pages-instructor/instructor-course-details-page/__snapshots__/instructor-course-details-page.component.spec.ts.snap

```diff
@@ -45,7 +46,7 @@ exports[`InstructorCourseDetailsPageComponent should snap with a course with one
               <p
                 class="form-control-static"
               >
-                CS101
```

Comment:
Why does this line get disappeared? Forget to update the test case?

Reply:
You're right. Updated the test case